mlrun/api/crud/client_spec.py

```diff
@@ -31,6 +31,7 @@ def get_client_spec(self):
             mpijob_crd_version=mpijob_crd_version,
             ui_url=config.resolve_ui_url(),
             artifact_path=config.artifact_path,
+            default_feature_store_data_prefix=config.feature_store.data_prefixes.default,
```

Comment:
For UI purposes, you need to place these fields in the `frontend-spec`, not the `client-spec`. The client-spec is for the SDK usage (processed by `httpdb` when connecting). It is fine to include these in the `client-spec` as well, for sdk usage, but it won't solve the issue this was meant to solve.

Reply:
Spoke with @theSaarco offline, we shall leave these in the client-spec as they might be useful as default paths but will add them to frontend spec as well